core/tests/performance_framework/perf_domain.py

```diff
@@ -144,3 +144,33 @@ def get_dom_ready_time_millisecs(self):
     def get_request_time_millisecs(self):
         """Returns the total request time (in milliseconds)."""
         return self._get_duration_millisecs('responseEnd', 'requestStart')
+
+
+class MultiplePageSessionMetrics(object):
+    """Domain object for multiple PageSessionMetrics.
```

Comment:
Perhaps explain the role of this class (e.g. to provide average statistics to reduce the noise that may be collected during recording time), or something along those lines.